poll/poll.py

```diff
@@ -140,53 +142,60 @@ def gather_items(self, data, result, noun, field, image=True):
             source_items = []
             result['success'] = False
             result['errors'].append(
-                "'{0}' is not present, or not a JSON array.".format(field))
+                self.ugettext("'{0}' is not present, or not a JSON array.").format(field))
         else:
             source_items = data[field]
 
         # Make sure all components are present and clean them.
         for item in source_items:
             if not isinstance(item, dict):
                 result['success'] = False
+                # Translators: {noun} is either "Answer" or "Question". {item} identifies the answer or question.
                 result['errors'].append(
-                    "{0} {1} not a javascript object!".format(noun, item))
+                    self.ugettext("{noun} {item} not a javascript object!").format(noun=noun, item=item))
                 continue
             key = item.get('key', '').strip()
             if not key:
                 result['success'] = False
+                # Translators: {noun} is either "Answer" or "Question". {item} identifies the answer or question.
                 result['errors'].append(
-                    "{0} {1} contains no key.".format(noun, item))
+                    self.ugettext("{noun} {item} contains no key.").format(noun=noun, item=item))
             image_link = item.get('img', '').strip()
             image_alt = item.get('img_alt', '').strip()
             label = item.get('label', '').strip()
             if not label:
                 if image and not image_link:
                     result['success'] = False
+                    # translators: {noun} is either "Answer" or "Question". {noun_lower} is the lower case of either
+                    # Answer or Question.
```

Comment:
@asadiqbal08 Capitalize "translators". Also, you can shorten the second sentence to "{noun_lower} is the lowercase version of {noun}. Same comment applies to the next translator comment in L179 below.